File: 1. Input, Processing and Output/exercise02/python/counting_characters.py
# Exercise 02 - Counting Characters

# Get input from the user
input_text = input('What is the input string? ')

# Perform calculations
count = len(input_text)
character = 'character' if count == 1 else 'characters'

# Format output
output_text = input_text + ' has ' + str(count) + ' ' + character + '.'

print(output_text)

File: 1. Input, Processing and Output/exercise02/java/CountingCharacters.java
import java.util.Scanner;

public class CountingCharacters {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        String text, messageOutput;

        System.out.print("What is the input string? ");
        text = scan.next();
        scan.close();

        messageOutput = String.format("%s has %d characters", text, text.length());

        System.out.println(messageOutput);

    }
}